sherpa-onnx/python/csrc/online-recognizer.cc

```diff
@@ -67,7 +67,7 @@ static void PybindOnlineRecognizerConfig(py::module *m) {
            py::arg("max_active_paths") = 4, py::arg("hotwords_file") = "",
            py::arg("hotwords_score") = 0, py::arg("blank_penalty") = 0.0,
            py::arg("temperature_scale") = 2.0, py::arg("rule_fsts") = "",
-           py::arg("rule_fars") = "")
+           py::arg("rule_fars") = "", py::arg("reset_encoder"))
```

Comment:
```suggestion
           py::arg("rule_fars") = "", py::arg("reset_encoder") = false)
```

Can you give it a default value?